### DISCUSSION TITLE
Progress stops updating on long print

### DISCUSSION THREAD

**Discussion by sblionel** [UnchartedBull/OctoDash#1882] [Q&A]
Using 2.2.0. Several times recently, on long prints (more than 24 hours), Octodash stops updating the print progress (percent done, time remaining, layer number) while the print continues. It is otherwise responsive - I can change the nozzle temperature, for instance. Octoprint shows the correct info. The print completes, Octodash still shows it has hours to go, and if I select Cancel I get an error saying it can't synchronize with Octoprint. See photo attached.

![IMG-2458 (1)](https://user-images.githubusercontent.com/30088263/120087362-2b1f0980-c0b5-11eb-8adb-aadc4ffb6fd8.jpg)

**Comment by eboston** [comment]
Did you post the correct picture?

**Reply by sblionel** [reply]
No - it resized wrong. Fixed.

**Comment by sblionel** [comment]
It did it again today - this time just at the start of the print. The display reads Printing - 0%, but the actual print is at 30%. Again, the display is responsive, and I can adjust temps, etc.

**Comment by UnchartedBull** [comment]
This is most likely due to the socket connection being dropped and is currently being investigated. It happens on some systems, on others it doesn't. Please try the following if you haven't already:

- Use `localhost` instead of an IP or `octopi.local`
- Ensure stable WiFi Connection of the Raspberry Pi

Does this usually happen after the same amount of time (i.e. always after 6 hours of being on?)

**Reply by sblionel** [reply]
The first few times it happened, it was after about 25 hours into the print - the Pi had been on for days. But the most recent time, it stopped updating as soon as the print started. The Pi has a great WiFi connection, but I used the IP address (pretty sure - it's printing now so I can't check), but will see if localhost helps.

I don't think I saw this at all before 2.2.0.

**Reply by UnchartedBull** [reply]
Thanks for the info! v2.2.0 changed the way that OctoDash is retrieving data from OctoPrint, so this error couldn't occur beforehand.
